sdcm/mgmt/common.py

```diff
@@ -180,3 +181,16 @@ class AgentBackupParameters(BaseModel):
     low_level_retries: Optional[int] = 20
 
     model_config = ConfigDict(arbitrary_types_allowed=False)
+
+
+def get_backup_size(mgr_cluster, task_id):
+    """
+    Returns the generated backup size of a given Manager backup Task.
+    """
+    res = mgr_cluster.sctool.run(cmd=f"progress {task_id} -c {mgr_cluster.id}",
+                                 parse_table_res=False)
+    match = re.search(r".+100% │ (.*?) │ ", res.stdout, re.MULTILINE)
```

Comment:
Please compile pattern separately and outside the function

Reply:
fixed